pkg/ccl/changefeedccl/cdctest/validator_helper.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2018 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+
+package cdctest
+
+import (
+	gojson "encoding/json"
+
+	"github.com/cockroachdb/cockroach/pkg/geo"
+	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
+	"github.com/cockroachdb/cockroach/pkg/util/json"
+	"github.com/cockroachdb/cockroach/pkg/workload/rand"
+)
+
+func extractFieldFromJSON(d json.JSON) (interface{}, error) {
```

Comment:
can you make this name more descriptive or add a comment about what it's doing? maybe like `attempts to coerce json data into a known go type, including geo` or something

Reply:
Done. Added a comment.